### DISCUSSION TITLE
Week 5- Exercise Feedback and Project Updates

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#115] [General]
Likes: 

- Javascript
I like how this training camp is structured. I think that all of the exercises found are very helpful. 
- HTML
The nature of these exercises is very helpful because they build upon each other through working on project forums. 
- Flexbox Froggy 
This one was super fun! I enjoyed seeing the actions of the frog as I tried out my code. 

Dislikes: 
- Javascript
Again with these exercises, I am missing tutorials. 
- HTML
This could use more instructions and the help isn't that helpful. 
- Flexbox Froggy 
I would like to see more walkthrough videos/tutorials. 
Changes: 
- Javascript
I would add more walkthroughs. 
- HTML
I would add finished registration forms for students to look at. 
- Flexbox Froggy 
I would add more interactive elements to the exercise as in more things to code out. 
Project Updates: 

- Portfolio: 
The HTML index would be the foundation of the website. I would then use CSS to make the site look better. I have already created my own portfolio in the past but it looked like something out of 1999. I don't think I would add JavaScript elements from what we have learned so far, I'm just not as familiar with it. 

- Online Game:
I would use CSS to code out the game and actions that the player could produce. This would be a harder project for me so I am mainly using this idea for a fun back up!

**Comment by metasean** [comment]
> HTML
> I would add finished registration forms for students to look at.

You should see the finished project just before the first step starts 
e.g. https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-1 should display:
<img width="1220" alt="Screen Shot 2023-06-13 at 6 45 01 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/7552754/6b02dd4c-6f03-416d-a779-e90a154db017">
